### ISSUE TITLE
Tests failing with 5.31.5

### ISSUE THREAD

**Issue by genehack** [pjcj/Devel--Cover#245]
This test is failing with 5.31.5: 

```
t/e2e/asignature.t .......... 1/1
#   Failed test 'output'
#   at blib/lib/Devel/Cover/Test.pm line 331.
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# | Elt|Got                                                                           | Elt|Expected                                                                      |
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# *  13|  'tests/signature  100.0   50.0    n/a  100.0   93.3                         *  13|  'tests/signature  100.0   50.0    n/a  100.0   95.8                         *
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# *  15|  'Total            100.0   50.0    n/a  100.0   93.3                         *  15|  'Total            100.0   50.0    n/a  100.0   95.8                         *
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# *  83|  '17                                      sub xx ($p, $q, $r, $s = 6, @t) {  *  83|  '17             1                    1   sub xx ($p, $q, $r, $s = 6, @t) {  *
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# *  85|  '18                                          $p + $q + $r + $s + @t\n       *  85|  '               1\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\s\n         *
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# |    |                                                                              *  87|  '               1                                                           *
# |    |                                                                              *  88|',                                                                            *
# |    |                                                                              *  89|  '               1                                                           *
# |    |                                                                              *  90|',                                                                            *
# |    |                                                                              *  91|  '               1                                                           *
# |    |                                                                              *  92|',                                                                            *
# |    |                                                                              *  93|  '               1                                                           *
# |    |                                                                              *  94|',                                                                            *
# |    |                                                                              *  95|  '               1                                                           *
# |    |                                                                              *  96|',                                                                            *
# |    |                                                                              *  97|  '18             1                            $p + $q + $r + $s + @t         *
# |    |                                                                              *  98|',                                                                            *
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# * 122|',                                                                            *    |                                                                              |
# * 123|  '- ----- ------------------                                                 *    |                                                                              |
# * 124|',                                                                            *    |                                                                              |
# * 125|  'BEGIN          1 tests/signature:13                                        *    |                                                                              |
# * 126|',                                                                            *    |                                                                              |
# * 127|  'BEGIN          1 tests/signature:14                                        *    |                                                                              |
# * 128|',                                                                            *    |                                                                              |
# * 129|  'BEGIN          1 tests/signature:15                                        *    |                                                                              |
# * 130|',                                                                            *    |                                                                              |
# * 131|  '                                                                           *    |                                                                              |
# * 132|',                                                                            *    |                                                                              |
# * 133|  '                                                                           *    |                                                                              |
# +----+------------------------------------------------------------------------------+----+------------------------------------------------------------------------------+
# Looks like you failed 1 test of 1.
t/e2e/asignature.t .......... Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests
```

```
Test Summary Report
-------------------
t/e2e/asignature.t        (Wstat: 256 Tests: 1 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=80, Tests=93, 45 wallclock secs ( 0.25 usr  0.08 sys + 37.58 cusr  5.45 csys = 43.36 CPU)
Result: FAIL
Failed 1/80 test programs. 1/93 subtests failed.
make: *** [Makefile:1253: test_dynamic] Error 255
```

```
% perl -V   
Summary of my perl5 (revision 5 version 31 subversion 5) configuration:

  Platform:
    osname=linux
    osvers=5.1.11-x86_64-linode127
    archname=x86_64-linux
    uname='linux klatch 5.1.11-x86_64-linode127 #1 smp preempt mon jun 17 21:18:26 utc 2019 x86_64 gnulinux '
    config_args='-Dprefix=/opt/plenv/versions/5.31.5 -de -Dversiononly -Dusedevel -A'eval:scriptdir=/opt/plenv/versions/5.31.5/bin''
    hint=recommended
    useposix=true
    d_sigaction=define
    useithreads=undef
    usemultiplicity=undef
    use64bitint=define
    use64bitall=define
    uselongdouble=undef
    usemymalloc=n
    default_inc_excludes_dot=define
    bincompat5005=undef
  Compiler:
    cc='cc'
    ccflags ='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2'
    optimize='-O2'
    cppflags='-fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include'
    ccversion=''
    gccversion='8.3.0'
    gccosandvers=''
    intsize=4
    longsize=8
    ptrsize=8
    doublesize=8
    byteorder=12345678
    doublekind=3
    d_longlong=define
    longlongsize=8
    d_longdbl=define
    longdblsize=16
    longdblkind=3
    ivtype='long'
    ivsize=8
    nvtype='double'
    nvsize=8
    Off_t='off_t'
    lseeksize=8
    alignbytes=8
    prototype=define
  Linker and Libraries:                                                                                                                                                  [8/14036]
    ld='cc'
    ldflags =' -fstack-protector-strong -L/usr/local/lib'
    libpth=/usr/local/lib /usr/lib/gcc/x86_64-linux-gnu/8/include-fixed /usr/include/x86_64-linux-gnu /usr/lib /lib/x86_64-linux-gnu /lib/../lib /usr/lib/x86_64-linux-gnu /usr/li
b/../lib /lib
    libs=-lpthread -lnsl -lgdbm -ldl -lm -lcrypt -lutil -lc
    perllibs=-lpthread -lnsl -ldl -lm -lcrypt -lutil -lc
    libc=libc-2.28.so
    so=so
    useshrplib=false
    libperl=libperl.a
    gnulibc_version='2.28'
  Dynamic Linking:
    dlsrc=dl_dlopen.xs
    dlext=so
    d_dlsymun=undef
    ccdlflags='-Wl,-E'
    cccdlflags='-fPIC'
    lddlflags='-shared -O2 -L/usr/local/lib -fstack-protector-strong'


Characteristics of this binary (from libperl):
  Compile-time options:
    HAS_TIMES
    PERLIO_LAYERS
    PERL_COPY_ON_WRITE
    PERL_DONT_CREATE_GVSV
    PERL_MALLOC_WRAP
    PERL_OP_PARENT
    PERL_PRESERVE_IVUV
    PERL_USE_DEVEL
    USE_64_BIT_ALL
    USE_64_BIT_INT
    USE_LARGE_FILES
    USE_LOCALE
    USE_LOCALE_COLLATE
    USE_LOCALE_CTYPE
    USE_LOCALE_NUMERIC
    USE_LOCALE_TIME
    USE_PERLIO
    USE_PERL_ATOF
  Locally applied patches:
    Devel::PatchPerl 1.52
  Built under linux
  Compiled at Oct 20 2019 12:29:07
  %ENV:
    PERL_CPANM_OPT="--skip-installed --prompt"
    PERL_MM_USE_DEFAULT="1"
    PERL_USE_UNSAFE_INC="1"
  @INC:
    /opt/plenv/versions/5.31.5/lib/perl5/site_perl/5.31.5/x86_64-linux
    /opt/plenv/versions/5.31.5/lib/perl5/site_perl/5.31.5
    /opt/plenv/versions/5.31.5/lib/perl5/5.31.5/x86_64-linux
    /opt/plenv/versions/5.31.5/lib/perl5/5.31.5
    .
```

**Comment by pjcj** [comment]
See 4df857782a14d0973f58f6a629019d29259b2aad

**Comment by pjcj** [comment]
Thanks very much for reporting this. It has been fixed in 164759f in the 1.34 release.

See also #260
